src/main/scala/org/allenai/deep_qa/experiments/datasets/OmnibusDa.scala

```diff
@@ -0,0 +1,70 @@
+package org.allenai.deep_qa.experiments.datasets
```

Comment:
The alternative is to have lots of files in `datasets`.  I guess that's not a huge problem.  Pick your poison...